Reneiski_Homework1/ReneiskiSelenium11/PageObjects/CheckBoxPage/CheckBoxPage.cs

```diff
@@ -0,0 +1,23 @@
+﻿using OpenQA.Selenium;
+
+namespace ReneiskiSelenium11.PageObjects.CheckBoxPage
+{
+    public class CheckBoxPage : BasePage
+    {
+        public CheckBoxPage(IWebDriver _driver) : base(_driver)
+        {
+        }
+
+        const string expectedText = "You have selected :\r\nhome\r\ndesktop\r\nnotes\r\ncommands\r\ndocuments\r\nworkspace\r\nreact\r\nangular\r\nveu\r\noffice\r\npublic\r\nprivate\r\nclassified\r\ngeneral\r\ndownloads\r\nwordFile\r\nexcelFile";
+
+        private IWebElement result => _driver.FindElement(By.Id("result"));
+
+        public void ClickCheckBox() => FindElementByXPathClass("rct-icon rct-icon-uncheck").Click();
+
+        public bool GetAttributeCheckBox() => FindElementByXPathClass("rct-icon rct-icon-check").GetAttribute("class").Contains("check");
```

Comment:
I would say that it's better to implement wrapper for IWebElement (like MyWebElement)  where you will store all the logic about clicking, finding element etc

Reply:
Fixed